.yapfignore

```diff
@@ -0,0 +1 @@
+hoomd/extern
```

Comment:
You could delete this file and control the exclusion via pre-commit's config if desired. Just a tradeoff between "works without pre-commit" and "having many files in the repo root."

Reply:
done.